src/test/java/teammates/test/pageobjects/AdminActivityLogPage.java

```diff
@@ -33,7 +33,12 @@ public String getPersonInfoOfFirstEntry(){
     public void clickViewActionsButtonOfFirstEntry(){
         
         WebElement table = browser.driver.findElement(By.id("logsTable"));
-        WebElement tableRow = table.findElements(By.tagName("tr")).get(1);
+        WebElement tableRow;
+        try {
+            tableRow = table.findElements(By.tagName("tr")).get(1);
+        } catch (IndexOutOfBoundsException onlyoneentry) {
+            tableRow = table.findElements(By.tagName("tr")).get(0);
+        }
```

Comment:
Try not to use exceptions to control the normal workflow. may be check the result size first?